hadoop-tools/hadoop-azure/dev-support/testrun-scripts/runtests.sh

```diff
@@ -117,141 +107,154 @@ runNonHNSOAuthDFSIngressBlobTest()
       ;;
-   esac
-done
+    SET_OR_CHANGE_TEST_ACCOUNT)
+      runTest=false
+      cleanUpTestContainers=false
+      accountSettingsFile="src/test/resources/azure-auth-keys.xml"
+      if [[ ! -f "$accountSettingsFile" ]]; then
+        logOutput "No settings present. Creating new settings file ($accountSettingsFile) from template"
+        cp src/test/resources/azure-auth-keys.xml.template $accountSettingsFile
+      fi
 
+      vi $accountSettingsFile
+      exit 0
+      break
+      ;;
+    PRINT_LOG4J_LOG_PATHS_FROM_LAST_RUN)
+      runTest=false
+      cleanUpTestContainers=false
+      logFilePaths=/tmp/logPaths
+      find target/ -name "*output.txt" >$logFilePaths
+      logOutput "$(cat $logFilePaths)"
+      rm $logFilePaths
+      exit 0
+      break
+      ;;
+    *)
+      logOutput "ERROR: Invalid selection"
+      ;;
+    esac
+  done
 
-## SECTION: COMBINATION DEFINITIONS AND TRIGGER
+  ## SECTION: COMBINATION DEFINITIONS AND TRIGGER
 
-echo ' '
-echo 'Set the active test combination to run the action:'
-select combo in HNS-OAuth-DFS HNS-SharedKey-DFS NonHNS-SharedKey-DFS AppendBlob-HNS-OAuth-DFS NonHNS-SharedKey-Blob NonHNS-OAuth-DFS NonHNS-OAuth-Blob AppendBlob-NonHNS-OAuth-Blob HNS-Oauth-DFS-IngressBlob NonHNS-Oauth-DFS-IngressBlob AllCombinationsTestRun Quit
-do
-   case $combo in
-      HNS-OAuth-DFS)
-         runHNSOAuthDFSTest
-         break
-         ;;
-      HNS-SharedKey-DFS)
-         runHNSSharedKeyDFSTest
-         break
-         ;;
-      NonHNS-SharedKey-DFS)
-         runNonHNSSharedKeyDFSTest
-         break
-         ;;
-       AppendBlob-HNS-OAuth-DFS)
-         runAppendBlobHNSOAuthDFSTest
-         break
-         ;;
-       NonHNS-SharedKey-Blob)
-         runNonHNSSharedKeyBlobTest
-         break
-         ;;
-        NonHNS-OAuth-DFS)
-         runNonHNSOAuthDFSTest
-         break
-         ;;
-        NonHNS-OAuth-Blob)
-         runNonHNSOAuthBlobTest
-         break
-         ;;
-        AppendBlob-NonHNS-OAuth-Blob)
-         runAppendBlobNonHNSOAuthBlobTest
-         break
-         ;;
-        HNS-Oauth-DFS-IngressBlob)
-         runHNSOAuthDFSIngressBlobTest
-         break
-         ;;
-        NonHNS-Oauth-DFS-IngressBlob)
-         runNonHNSOAuthDFSIngressBlobTest
-         break
-         ;;
-      AllCombinationsTestRun)
-        if [ $runTest == false ]
-        then
-          logOutput "ERROR: Invalid selection for SET_ACTIVE_TEST_CONFIG. This is applicable only for RUN_TEST."
-          break
-        fi
-        runHNSOAuthDFSTest
-        runHNSSharedKeyDFSTest
-        runNonHNSSharedKeyDFSTest
-        runAppendBlobHNSOAuthDFSTest
-        runNonHNSSharedKeyBlobTest
-        runNonHNSOAuthDFSTest
-        runNonHNSOAuthBlobTest
-        runAppendBlobNonHNSOAuthBlobTest
-        runHNSOAuthDFSIngressBlobTest
-        runNonHNSOAuthDFSIngressBlobTest
-         break
-         ;;
-      Quit)
-         exit 0
-         ;;
-      *) logOutput "ERROR: Invalid selection"
+  echo ' '
```

Comment:
Same dont see any change between new code and old one for this selection. minimizing the diffs would be better for backporting